api/handlers.go

```diff
@@ -86,6 +100,27 @@ func (sl *skylink) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// blocklistGET returns a list of blocked hashes and associated tags
+func (api *API) blocklistGET(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	blocked, err := api.staticDB.BlockedHashes()
+	if err != nil {
+		skyapi.WriteError(w, skyapi.Error{err.Error()}, http.StatusInternalServerError)
+		return
+	}
+
+	hashes := make([]BlockedHash, len(blocked))
+	for i, bh := range blocked {
+		hashes[i] = BlockedHash{
+			Hash: bh.Hash.Hash,
+			Tags: bh.Tags,
+		}
+	}
+	skyapi.WriteJSON(w, BlocklistGET{
+		Hashes:  hashes,
+		Created: time.Now().UTC(),
```

Comment:
Is there any need for that? The client should now when it received the response. We are not really using this timestamp for anything right?

Reply:
yeah we're not, removed it